pkg/drivers/kic/oci/oci.go

```diff
@@ -69,6 +69,25 @@ func DeleteContainersByLabel(ociBin string, label string) []error {
 	return deleteErrs
 }
 
+// DeleteContainer deletes a container by ID or Name
+func DeleteContainer(ociBin string, name string) error {
+	if err := PointToHostDockerDaemon(); err != nil {
+		return errors.Wrap(err, "point host docker daemon")
+	}
+	_, err := ContainerStatus(ociBin, name)
+	// only try to delete if docker/podman inspect returns
```

Comment:
This comment is no longer accurate. Perhaps we should just remove the Status check?